src/index.html

```diff
@@ -4,9 +4,323 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo-moyo">
```

Comment:
```suggestion
        <img
          class="header__logo"
          src="./images/logo.png"
          alt="logo-moyo"
        >
```
Closing tag should be just under the opening